mayastor/src/core/env.rs

```diff
@@ -601,6 +608,17 @@ impl MayastorEnvironment {
         cfg.apply();
     }
 
+    /// load the pool config file.
+    fn load_pool_config(&self) -> Option<PoolConfig> {
```

Comment:
This should not fail mayastor to start.

Reply:
Now issues a warning in the log rather than trigger a panic